docs/current.settings.yaml

```diff
@@ -1,4 +1,4 @@
-config_version: 2
+config_version: 000
```

Comment:
well... it's not 1 and it is not 2... :)

should we change to using dot versions or `2-beta`?